### DISCUSSION TITLE
Welcome to single-file-cli Discussions!

### DISCUSSION THREAD

**Discussion by gildas-lormeau** [gildas-lormeau/single-file-cli#72] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Use-SF** [comment]
Hi, I am new here. I'm from Germany. I have installed "SingleFile" under Firefox and under "Edge". No problems, but since some days, if I use "Edge" the html-file, produced by SingleFile, is stored as a cryptic one, e.g. "21944ce5b-d6d8-4a49-9e27-235cdf1592a9.htm", no clear name anymore. Under Firefox the given name is fine. I have tried to change the settings. No chance. So, perhapy anyone has an idea, to solve my problem.
Best regards
Use-SF

**Reply by gildas-lormeau** [reply]
This is a known issue in Chrome, see the 2nd item in the "Chromium-based browsers" section for more info here: https://github.com/gildas-lormeau/SingleFile/blob/master/known-issues.md
